.github/workflows/run-terra-featured-workspace-tests.yaml

```diff
@@ -0,0 +1,242 @@
+name: Run Terra Featured Workspace Tests
+on:
+  schedule:
+    - cron: '0 0 */14 * *' # Every 2 weeks on Sunday at 00:00 UTC
+  workflow_dispatch:
+    inputs:
+      mute_notifications:
+        description: 'Mute notifications'
+        default: 'true'
+        required: false
+      troubleshoot:
+        description: 'Enable troubleshooting'
+        default: 'true'
+        required: false
+
+env:
+  FORCE_JAVASCRIPT_ACTIONS_TO_NODE20: true
+  GCP_PROJECT_ID: "terra-featured-workspace-tests"
+  GCP_SA_EMAIL: "terra-featured-workspace-tests@terra-featured-workspace-tests.iam.gserviceaccount.com"
+jobs:
+  run-featured-workspace-tests-batch-1:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Generate CURRENT_DATE
+        id: date
+        run: |
+          echo "CURRENT_DATE=$(date +%Y-%m-%d-%H-%M-%S)" >> $GITHUB_ENV
+
+      - name: Authenticate to GCP
+        uses: google-github-actions/auth@v2
+        with:
+          project_id: ${{ env.GCP_PROJECT_ID }}
+          workload_identity_provider: "projects/953233968740/locations/global/workloadIdentityPools/github-actions-pool/providers/github-actions-provider"
+          service_account: ${{ env.GCP_SA_EMAIL }}
+
+      - id: secrets
+        uses: google-github-actions/get-secretmanager-secrets@v2
+        with:
+          secrets: |-
+            sendgrid_key:${{ env.GCP_PROJECT_ID }}/sendgrid_key
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          pip install -r requirements.txt
+
+      - name: Run Terra Featured Workspace Tests - Batch 1
+        env:
+          SENDGRID_KEY: ${{ steps.secrets.outputs.sendgrid_key }}
+        run: |
+          set -e
+          REPORT_NAME="master_report_batch_1${CURRENT_DATE}.html"
```

Comment:
for each of these you probably want to put a _ after the batch number.
You're getting report names like "terra-featured-workspace-tests-reports/fw_reports/master_report_batch_12025-02-05-00-05-40.html" right now - which makes sense if you think about or look at the code, but is confusing at first glance - looks like the batch # is part of the timestamp

Reply:
yes...that needs to be changed